test/drivers/mysql.test.js

```diff
@@ -0,0 +1,47 @@
+const { expect } = require('chai');
+const connection = require('../../knexfile');
+// eslint-disable-next-line import/order
+const knex = require('knex')(connection.development);
+const MySQLDriver = require('../../lib/dbDriver/mysql');
+
+describe('MySQL/Knex Persistance', () => {
+  before((callback) => {
+    knex.migrate.latest().then(() => {
+      callback();
+    });
+  });
+
+  it('@integration: addLink stores an item to the db', async () => {
+    const driver = new MySQLDriver();
+    driver.addLink('sqlIntegration', 'MySQLIntegration', 'integrations.com');
+
+    await knex.select('*')
+      .from('project_links')
+      .where({ project_id: 'sqlIntegration' })
+      .then(rows => expect(rows[0].label).to.equal('MySQLIntegration'));
+
+    await knex.select('label')
+      .from('project_links')
+      .where({ label: 'MySQLIntegration' })
+      .del();
+  });
+
+  it('Knex stores an item to the db', async () => {
+    knex('project_links')
+      .insert({ project_id: 'knexTesting', label: 'knexStoreTest', url: 'testInput.com' })
+      .then()
```

Comment:
why are we calling 'then' here?

Reply:
I'm not super confident we need these tests since we have the `ProjectLink.save` tests above. What do you think?

Reply:
I thought about this, and I mentioned to @ColorfulCodes that you may bring this up. 

My idea was that since the `addLink` serves as an integration test, we'd need a unit test on the actual knex persistence. But it's literally doing the same thing.. So it's unnecessary.